papers/orhan_eroglu/00_geocat.rst

```diff
@@ -162,23 +160,23 @@ GeoCAT-comp (and GeoCAT-f2py)
 
 GeoCAT-comp is the computational component of the GeoCAT project as can be seen in
 Figure :ref:`fig3structure`, and it houses implementations of geoscience data analysis
-functions. Novel research and development is conducted for analyzing structured as
-well as unstructured grid data from various research fields such as climate,
+functions. Novel research and development is conducted for analyzing structured and
+unstructured grid data from various research fields including climate,
 weather, atmosphere, ocean, etc. In addition, some of the functionalities of
-GeoCAT-comp are inspired/reimplemented from NCAR Command Language (NCL).
+GeoCAT-comp are modernized versions of functions from NCAR Command Language (NCL).
 
 Spherical harmonics (both decomposition and recomposition as well as area
 weighting); Fourier transforms such as band-block, band-pass, low-pass, and
-high-pass; meteorological variables such as relative humidity, dewpoint temperature,
-heat index, saturation vapure pressure, etc.; climatology functions such as climate
+high-pass; meteorological variables such as relative humidity, dew point temperature,
+heat index, saturation vapor pressure, etc.; climatology functions such as climate
 average over multiple years, daily/monthly/seasonal averages, as well as anomalies;
 regridding of curvilinear grid to rectilinear grid, unstructured grid to rectilinear
 grid, curvilinear grid to unstructured grid, and vice versa; interpolation methods
 such as bilinear interpolation of a rectilinear to another rectilinear grid,
 hybrid-sigma levels to isobaric levels, and sigma to hybrid coordinates; empirical
 orthogonal function (EOF) analysis; are some of the functions that are made
 available through GeoCAT-comp. GeoCAT-comp documentation :cite:`geocatcomprtd`
-provides signatures and descriptions of these functions as well as a line to the
+provides signatures and descriptions of these functions as well as links to
 usage examples.
```

Comment:
I think this could be worded better. Is there a possibility the first sentence could be turned in to bullet points? At the very least I would suggest re-ordering the sentence to start with "GeoCAT-comp provides ..." instead of ending with it,

Reply:
I reordered this entire paragraph by moving the last two sentences to the beginning then listing all those functions as bullet points. Thanks!